.dockerignore

```diff
@@ -1 +1,2 @@
 .git
+test
```

Comment:
I don't understand; why is the `test` directory excluded? This would seem to exclude the possibility of running the integration tests from Docker and doesn't add anything significant as the tests aren't included in the final Docker image anyway.

Reply:
I'm not sure. @rodrigovalin could you elaborate? (should've asked earlier)

Reply:
The question for me is not why is the directory excluded, but why is it included? Are you planning on running the tests inside the Docker image? Having smaller contexts make the docker build process faster

Reply:
I definitely agree that it's a good idea to keep the docker context small but I believe that it's more important to keep it similar to the gitignore. That ensures that the resultant docker image is consistent with a local development environment.

With regards to this specific case, the test directory is small and I can definitely foresee running the tests from within a docker image. Removing it also adds a degree of inconsistency, whereby we have some subset of the tests in the docker image, but not all. If we decide that we don't want any tests in the docker image then we should figure out how to do that rather than removing just some.